utilities/jfr/jenkinsfile-runner.inc

```diff
@@ -0,0 +1,18 @@
+# function to run a docker image
+# $1: tag of the docker image
+# $2: Path to Jenkinsfile
+run_jfr_docker_image() {
+
+	if [ "$#" -gt 1 ]
+	then
+        docker run --rm -v $2:/workspace/Jenkinsfile $1
+		return 0
+	else
+		echo "Error. Missing tag for the docker image to run"
```

Comment:
I think this first message can be removed?

Reply:
You're right. I changed a previous error messages, and I missed this one